go.mod

```diff
@@ -119,6 +120,9 @@ require (
 	github.com/spf13/afero v1.1.2 // indirect
 	github.com/spf13/jwalterweatherman v1.0.0 // indirect
 	github.com/subosito/gotenv v1.2.0 // indirect
+	github.com/tidwall/gjson v1.14.4 // indirect
```

Comment:
we should clean this up, we are not adding any dependency as part of this change as far as I know.